hook.php

```diff
@@ -98,7 +99,7 @@ function plugin_mreporting_install()
     $query_display_pref = "SELECT id
       FROM glpi_displaypreferences
       WHERE itemtype = 'PluginMreportingConfig'";
-    $res_display_pref = $DB->query($query_display_pref);
+    $res_display_pref = $DB->doQuery($query_display_pref);
```

Comment:
`doQuery()` has appeared in GLPI 10.0.11, so you need to change the value of `PLUGIN_MREPORTING_MIN_GLPI` in `setup.php`.

Reply:
In `setup.php`, you must have _(I can't comment on the right line directly)_:
```
define('PLUGIN_MREPORTING_MIN_GLPI', '10.0.11');
```